index.js

```diff
@@ -5,8 +5,8 @@ var fs = require('fs');
 var glob = require('glob');
 var path = require('path');
 
-var LINT_CONFIG = '// Generated by Peasant\n{\n  "extends": "airbnb/base"\n}\n';
-
+var LINT_CONFIG = '// Generated by Peasant\n{\n  "extends": "airbnb/base",\n  "env": {\n    "mocha": true\n  }\n}\n';
+var BABEL_CONFIG = '// Generated by Peasant\n{\n  "presets": ["latest"],\n  "plugins": ["transform-runtime"]\n}\n';
```

Comment:
For the future, it would be nice to make the runtime optional. It's really only needed for some new types and generator-based features like `async`/`await` in browsers or old Node.js.